src/main/java/de/zalando/zmon/dataservice/config/LightStepConfig.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.zmon.dataservice.config;
+
+import io.opentracing.Tracer;
+import com.lightstep.tracer.jre.JRETracer;
+import com.lightstep.tracer.shared.Options;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+
+public class LightStepConfig implements OpenTracerConfig {
+
+    private String lightStepHost = "localhost";
+    private int lightStepPort = 80;
+    private String lightStepAccessToken = "";
+    private String serviceName;
+
+    private final Logger logger = LoggerFactory.getLogger(OpenTracerConfig.class);
+
+    public Tracer generateTracer() {
+        Options opts = null;
+        try {
+            opts = new Options.OptionsBuilder()
+                    .withAccessToken(lightStepAccessToken)
+                    .withCollectorHost(lightStepHost)
+                    .withCollectorPort(lightStepPort)
+                    .withComponentName(serviceName)
+                    .build();
+        } catch (MalformedURLException e) {
```

Comment:
Doesn't seem to catch all possible misconfigurations. This is the only checked exception for the Hostname. Can we do better validating ports and/or other properties? Is an empty AccessToken valid?

Reply:
As of now I do not have any idea what misconfigurations can happen for lightstep...I can only test it once i have some infra setup for lightstep...so what I intended was to have a placeholder in the code for lightstep and activate this section of the code when we start testing lightstep...till then whenever the config is set to lightstep..i would return a noop tracer